pkg/glcode/lookup.go

```diff
@@ -0,0 +1,319 @@
+// Copyright 2019 The Moov Authors
+// Use of this source code is governed by an Apache License
+// license that can be found in the LICENSE file.
+
+// Generated on 2019-01-21T12:28:38Z by adam, any modifications will be overwritten
+package glcode
+
+var glCodes = map[string]string{
+	"0001": "Estimated cash receipts",
```

Comment:
I think having exported consts for each of these would be useful? That way paygate could reference `glcode.EstimatedCashReceipts`. Thoughts?

Reply:
Generating names for `Commingled Trust Funds Unamortized Premiums on Investments (SIB Only)` might be more difficult though..